Presentation/Patron/PatronOptions.cs

```diff
@@ -97,12 +106,13 @@ private async Task DeletePatron()
         Header.AppHeader();
         AnsiConsole.MarkupLine("[bold yellow]Deleted a patron[/]");
 
-        var patronId = AnsiConsole.Ask<Guid>("Enter the patron id: ");
+        long patronMembershipNumber = AnsiConsole.Ask<long>("Enter the patron member number: ");
```

Comment:
Can you make the Pattern selection the same as the one for editing?

To avoid situations like this and make the selection easier and faster:
![Screenshot from 2024-08-31 08-23-26](https://github.com/user-attachments/assets/e1a22cb6-ed74-4266-929e-20b029c03fb1)

![Screenshot from 2024-08-31 08-22-48](https://github.com/user-attachments/assets/a29adc5b-459b-40c8-a222-28be09ef4302)

Reply:
done